README.md

```diff
@@ -7,667 +7,73 @@
-```
-
-[Back to top](#top) / [Back to quick-start TOC](#qs-tutorial)
-
-<a id="run-mlrun-service"></a>
-### Running an MLRun Service
-
-An MLRun service is a web service that manages an MLRun database for tracking and logging MLRun run information, and exposes an HTTP API for working with the database and performing MLRun operations.
-
-You can create and run an MLRun service by using either of the following methods:
-- [Using Docker](#run-mlrun-service-docker)
-- [Using the MLRun CLI](#run-mlrun-service-cli)
-
-> **Note:** For both methods, you can optionally configure the service port and/or directory path by setting the `MLRUN_httpdb__port` and `MLRUN_httpdb__dirpath` environment variables instead of the respective run parameters or CLI options.
-
-<a id="run-mlrun-service-cli"></a>
-#### Using the MLRun CLI to Run an MLRun Service
-
-Use the `db` command of the MLRun CLI (`mlrun`) to create and run an instance of the MLRun service from the command line:
-```sh
-mlrun db [OPTIONS]
-```
-
-To see the supported options, run `mlrun db --help`:
-```
-Options:
-  -p, --port INTEGER  HTTP port for serving the API
-  -d, --dirpath TEXT  Path to the MLRun service directory
-```
+MLRun is composed of the following layers:
 
+- **[Feature Store](./docs/feature-store/feature-store.html)** &mdash; collects, prepares, catalogs, and serves data features for development (offline) and real-time (online) usage for real-time and batch data. See also 
+[Feature store: data ingestion](./docs/feature-store/feature-store-data-ingestion) and [Feature store: data retrieval](./docs/feature-store/feature-store-data-retrieval), as well as the [Feature Store tutorials](./docs/feature-store/feature-store-tutorials).
+- **[ML CI/CD pipeline](.docs//projects/ci-integration)** &mdash; automatically trains, tests, optimizes, and deploys or updates models using a snapshot of the production 
+data (generated by the feature store) and code from the source control (Git).
+- **[Real-Time Serving Pipeline](./docs/serving/serving-graph)** &mdash; Rapid deployment of scalable data and ML pipelines using real-time serverless technology, including 
+the API handling, data preparation/enrichment, [model serving](https://docs.mlrun.org/en/latest/serving/build-graph-model-serving.html), ensembles, driving and measuring actions, etc.
+- **[Real-Time monitoring and retraining](.docs//model_monitoring/index)** &mdash; monitors data, models, and production components and provides a feedback loop for exploring production data, identifying drift, alerting on anomalies or data quality issues, triggering re-training jobs, measuring business impact, etc.
+
+## Get started
+
+It's easy to start using MLRun: 
+1. Install MLRun using one of [over Kubernetes Cluster](https://docs.mlrun.org/en/latest/install/kubernetes.html) or [locally using Docker](https://docs.mlrun.org/en/latest/install/local-docker.html).<br>
```

Comment:
Install MLRun [locally using Docker] or [over Kubernetes Cluster]. Alternatively, you can use [Iguazio's managed MLRun service]